### PR TITLE
Give the Cloud VPC section some love

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

This PR:
* Moves all the Cloud VPC content on to a single page
* Puts the existing content into the new structure (including procedures)
* Gives everything a good copy-edit.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

**Comment by Loquacity** [comment]
image

File: cloud/vpc-peering.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+## Create a new VPC in Timescale Cloud
+To begin, you need to create a new VPC in the Timescale Cloud console.
+
+<procedure>
+
+### Creating a new VPC in Timescale Cloud
+1.  [Log in to your Timescale Cloud account][cloud-login] and navigate to
+    the `VPC` section.
+1.  Click `Create VPC`.
+1.  In the `Create a VPC` dialog, type a name for your new VPC, and provide an
```
Comment (clarkbw) [inline cloud/vpc-peering.md:43]:
```suggestion
1.  In the `Create a VPC` dialog, choose the region for your VPC that matches the region of the service you plan to attach to. change the name for your new VPC, and provide an
```

I'm not sure if this should be written out into sub steps

File: cloud/vpc-peering.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+<procedure>
+
+### Creating a new VPC in Timescale Cloud
+1.  [Log in to your Timescale Cloud account][cloud-login] and navigate to
+    the `VPC` section.
+1.  Click `Create VPC`.
+1.  In the `Create a VPC` dialog:
+    *   Type a name for your new VPC that matcghes the region of the service you
+        want to attach it to.
```
Comment (clarkbw) [inline cloud/vpc-peering.md:45]:
```suggestion
    *   Type a name for your new VPC and select the region that matches the region of the service you
        want to attach it to.
```

File: cloud/vpc-peering.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
+# VPC peering in Timescale Cloud
+Timescale Cloud allows you to create a virtual private cloud (VPC) network
+between an external cloud provider and your Timescale Cloud services. This
+allows you to isolate your Timescale Cloud services so that they are only
+accessible using your external cloud account, and is useful if you need to
+improve security through a reduction in the potential attach vector surface.
+
+When you have VPC peering set up in your external cloud provider, you can create
+and configure your VPC peering connections in the Timnescale Cloud console.
```
Comment (clarkbw) [inline cloud/vpc-peering.md:9]:
```suggestion
and configure your VPC peering connections in the Timescale Cloud console.
```

**Review by clarkbw — APPROVED** [review]
Looks good! Suggested a few small updates

**Comment by Loquacity** [comment]
🚀

**Pull request merged**